config/staging.json

```diff
@@ -26,7 +26,6 @@
     "appId": "saturnnonprod-O9lUP",
     "apiKey": "AQEBBAEkx4iE2KxNyI7Wx08EwU1ycTM7E4FMSmaibbMUQxNU6uQvuAJt7fyABAtFYSYfgEE"
   },
-  "azurePreviewGroup": true,
```

Comment:
For completeness, also remove from https://github.com/broadinstitute/terra-helmfile/blob/51adddd811b00a5c7d24c0c79ebb59ceb0562979/values/app/terraui/bee.yaml.gotmpl#L29.

Reply:
Thanks! Forgot about that one

Reply:
https://github.com/broadinstitute/terra-helmfile/pull/4602